packages/testing/config/storybook/main.js

```diff
@@ -46,7 +46,10 @@ const baseConfig = {
   stories: [
     `${importStatementPath(rwjsPaths.web.src)}/**/*.stories.{tsx,jsx,js}`,
   ],
-  addons: [config.web.a11y && '@storybook/addon-a11y'].filter(Boolean),
+  addons: [
+    config.web.a11y && '@storybook/addon-a11y',
+    '@storybook/addon-essentials',
```

Comment:
```suggestion
    '@storybook/addon-essentials',
    config.web.a11y && '@storybook/addon-a11y',
```

Reply:
IIRC the order of declaring the `addons`, does matter sometimes.

My suggestion might not be necessary here, just want to call it out in case someone who knows more about add-on bootstrap logic can confirm / deny ( @shilman ? )

Reply:
Just did a quick search and I think the order does matter: https://github.com/storybookjs/storybook/issues/11442

Code that checks: https://github.com/storybookjs/storybook/blob/239d0fc7d256e4e311b141278ba7b5d80d625dfc/lib/core-common/src/utils/check-addon-order.ts#L37-L47

So this suggestion is valid.

Reply:
Thanks for this ! :smile: